benchopt/plotting/html/templates/benchmark.mako.html

```diff
@@ -145,7 +147,7 @@ <h1>
       % if disp_sub:
         <!--Case where sub infos are available but somehow main infos are not-->
         % if disp_button:
-          <button onclick="displaymore(this,${idx_res})" id=${"btn_subinfo"+str(idx_res)} class="button buttoncent" style="float:right;">
+          <button onclick="displayMore(this,${idx_res})" id=${"btn_subinfo"+str(idx_res)} class="button buttoncent" style="float:right;">
```

Comment:
should we also use the `click` method or would this complicate the code too much?

Reply:
Maybe with something like `data-index` to store the `idx_res`, as describe [here](https://stackoverflow.com/a/53047367/2642845).

Reply:
Done in the latest commit (and we were already using the `data-?` attributes to handle the filenames :slightly_smiling_face: so the code is quite the same).